src/main/java/org/opentripplanner/api/mapping/StopMapper.java

```diff
@@ -27,11 +28,11 @@ public static ApiStop mapToApi(StopLocation domain, boolean extended) {
         api.lat = domain.getLat();
         api.lon = domain.getLon();
         api.code = domain.getCode();
-        api.name = domain.getName();
+        api.name = domain.getName().toString();
```

Comment:
Should we add a query parameter in the index API to be able to set the locale for these?

Reply:
Is it needed if plan is to remove index api in future?

Reply:
We can keep this as is.